app/views/provider_interface/_fields_for_permissions.html.erb

```diff
@@ -1,100 +1,85 @@
-<%= form_with(
```

Comment:
This is a rather unhelpful diff... I've removed the component and moved the fields_for section out into a partial.

I would have liked to put the fields_for bit into another component, but this doesn't seem to work. (Just replacing the partial with a `<%= render MyComponent.new(...) %>` is what I've tried)

Reply:
Looks like the issue I ran into is a known one for view_component: https://github.com/github/view_component/issues/241